### ISSUE TITLE
Extra pixels from one image to another

### ISSUE THREAD

**Issue by Camilochiang** [PRBonn/phenobench#3]
**TLDR**: There are object smallers than either witdh or height 1-3 pixel in the borders of the images, this make fitting boxes or masks technically wrong (as this pixels are not even green...).

**WARNING**:
This is a general warning regarding your dataset. Is a technicality and is up to you/the people who use the dataset how to interpretate. 

**issue**:
When working with `plant_instances` (Should be the same with other folders), there are masks who have either width or height of pixel 1-3 in the borders of the images. I inmagine that this is due the fact that the pictures were tilled to 1024 x 1024.  This make these pixels wrongly annotated due that as they are, there are not technically part of a plant, not even green and also make imposible to fit a box around (There are in the border, so the box will cover the pixels, leaving "no space for masks").

![image](https://github.com/PRBonn/phenobench/assets/48471579/077e7c67-74c1-44eb-a4b9-cb8b0085225e)

This is the case with images (when in a sorted list) of index:
`453, 736, 740, 1107, 1197, 1200, 1241, 1287, 1292, 1354, 1406`

**Comment by JaWeyl** [comment]
Dear Camilochiang, 

Thanks for highlighting this issue - we agree that these cases can occur due to patching the images.
As you already said it up to the user how to handle these cases when creating bounding boxes.

However, we emphasize that the user may use the provided visibility mask to filter instances that are too close to the border of an image. Particularly, this is also how we handle such cases during evaluation.

Best,

Jan
